official/resnet/ctl/ctl_imagenet_test.py

```diff
@@ -119,4 +116,6 @@ def test_end_to_end_2_gpu(self):
     )
 
 if __name__ == '__main__':
+  if not keras_common.is_v2_0():
+    tf.enable_v2_behavior()
```

Comment:
still suspicious of this.. ideally we just have one version of the test and run that with whatever TF is appropriate. i think for now this is nice becasue we are not running these tests with 2.0 yet. once we do, we can remove this from here and just skip the tests in 1.x?

Reply:
Sure. we can do that.